canvas_api/src/types/responses/conversations.rs

```diff
@@ -0,0 +1,33 @@
+use chrono::{DateTime, Utc};
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct Conversation {
+    pub id: u64,
+    pub subject: String,
+    pub last_message: String,
+    pub subscribed: bool,
+    pub private: bool,
+    pub starred: bool,
+    pub tags: Option<Vec<String>>,
+    pub message: Vec<ConversationMessage>
+}
+
+#[derive(Debug, Deserialize)]
+pub struct ConversationMessage {
+    pub id: u64,
+    pub body: String,
+    pub author_id: String,
```

Comment:
We're using `u64` for id's within this project.

(Except in config files, where they are i64 which is kinda jank)

```suggestion
    pub author_id: u64,
```